components/Repositories/YoloChart.tsx

```diff
@@ -0,0 +1,146 @@
+import * as HoverCard from "@radix-ui/react-hover-card";
+import Link from "next/link";
+import { FaArrowLeft, FaRegHandPeace } from "react-icons/fa6";
+import Skeleton from "react-loading-skeleton";
+import SkeletonWrapper from "components/atoms/SkeletonLoader/skeleton-wrapper";
+import Card from "components/atoms/Card/card";
+import Avatar from "components/atoms/Avatar/avatar";
+import HoverCardWrapper from "components/molecules/HoverCardWrapper/hover-card-wrapper";
+import { getAvatarByUsername } from "lib/utils/github";
+import Button from "components/shared/Button/button";
+
+type YoloChartProps = {
+  yoloStats: RepositoryYolo | undefined;
+  uniqueYoloCoders: Set<string> | undefined;
+  repository: string;
+  isLoading: boolean;
+  range: number;
+  backButtonOnClick?: () => void;
+  showHoverCards?: boolean;
+  className?: string;
+};
+
+export default function YoloChart({
+  yoloStats,
+  uniqueYoloCoders,
+  repository,
+  isLoading,
+  range,
+  backButtonOnClick,
+  showHoverCards,
+  className,
+}: YoloChartProps) {
+  return (
+    <Card className={`${className ?? ""} flex flex-col gap-4 w-full h-full items-center`}>
+      <div className="flex flex-col gap-2 items-start w-full">
+        {backButtonOnClick && (
+          <Button variant="text" onClick={backButtonOnClick} className="flex gap-2 text-sauced-orange">
+            <FaArrowLeft /> Back
+          </Button>
+        )}
+        <section className="flex flex-col lg:flex-row w-full items-start lg:items-center gap-4 lg:justify-between px-4">
+          <header className="flex w-full justify-between items-center">
+            <div className="flex gap-2 items-center">
+              <FaRegHandPeace className="text-xl" />
+              <h3 className="text-sm font-semibold xl:text-lg text-slate-800">YOLO Coders</h3>
+            </div>
+          </header>
+        </section>
+      </div>
+
+      <section className="w-full px-4 flex flex-col gap-4 text-xs xl:text-sm">
+        {isLoading ? (
+          <Skeleton height={32} />
+        ) : (
+          <p className="text-slate-500">
+            {uniqueYoloCoders && (
+              <>
+                <span className="font-semibold text-black">{`${uniqueYoloCoders.size} `}</span> contributor
+                {uniqueYoloCoders.size > 1 && "s"} {uniqueYoloCoders.size > 1 ? "have" : "has"} pushed{" "}
+                <span className="font-semibold text-black">{yoloStats?.num_yolo_pushed_commits} commits </span>
+                directly to the main branch in the last <span className="font-semibold text-black">{range}</span> days.
+              </>
+            )}
+          </p>
+        )}
+
+        <p>
+          <a href="https://docs.opensauced.pizza" target="_blank" className="text-sauced-orange underline">
+            Learn more
+          </a>{" "}
+          about why this could be a problem.
```

Comment:
```suggestion
          about why pull requests are a better way to contribute.
```
@BekahHW @isabensusan feedback on this?

I continue to stand on, "We have the ability to change how folks approach contributing in open source." I want to make sure we provide clear context as to why.

Reply:
I like the change! Definitely leans on more positive feedback, so +1

Reply:
+1

Reply:
+1 from me here as well: pushing directly to main without goo reason / process can be extremely risky with a bad force push or a nefarious maintainer embedding new malicious commits deep into the reflog.